lib/members.rb

```diff
@@ -1,17 +1,22 @@
 require 'open-uri'
 require 'open-uri/cached'
 require 'json'
-require_relative 'member_formatter'
+require 'field_serializer'
+require_relative 'member'
 
 OpenURI::Cache.cache_path = '.cache'
 
 class Members
+  include FieldSerializer
+
   def initialize(url)
     @url = url
   end
 
-  def to_a
-    format(all_members)
+  field :members do
```

Comment:
There's a bit of a mismatch here with what this class is representing. If this only providing information on lower house members, and all are available at the source, then _either_ the class should make that apparent, _or_ the method. (i.e. when we later want to also get the upper house, we should either be adding a new class or a new method, not modifying either of these).